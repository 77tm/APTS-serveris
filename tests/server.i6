500
1 378
1 472
1 435
1 44
1 143
1 125
2 95
2 174
2 99
3 48
3 92
3 340
4 112
4 65
4 23
5 352
5 10
5 200
6 254
6 97
6 467
7 371
7 211
7 464
8 458
8 31
8 144
9 90
9 217
9 53
10 215
10 335
10 143
11 161
11 390
11 405
12 349
12 461
12 167
13 366
13 367
13 144
14 394
14 239
14 422
15 324
15 148
15 439
16 322
16 381
16 404
17 61
17 338
17 30
18 269
18 104
18 56
19 418
19 445
19 212
20 136
20 50
20 196
21 232
21 34
21 429
22 343
22 338
22 456
23 228
23 299
23 247
24 248
24 128
24 212
25 409
25 344
25 112
26 456
26 129
26 328
27 141
27 87
27 269
28 265
28 171
28 82
29 145
29 158
29 490
30 222
30 433
30 336
31 454
31 409
31 459
32 73
32 115
32 338
33 45
33 143
33 13
34 222
34 166
34 272
35 39
35 228
35 460
36 330
36 21
36 3
37 80
37 351
37 138
38 87
38 453
38 39
39 338
39 92
39 247
40 458
40 153
40 213
41 419
41 227
41 60
42 154
42 412
42 412
43 182
43 113
43 370
44 34
44 390
44 21
45 364
45 175
45 287
46 499
46 208
46 175
47 201
47 223
47 475
48 291
48 497
48 118
49 235
49 374
49 146
50 400
50 310
50 91
51 475
51 77
51 174
52 364
52 453
52 273
53 366
53 92
53 278
54 139
54 429
54 107
55 245
55 191
55 362
56 221
56 425
56 392
57 271
57 381
57 404
58 146
58 254
58 191
59 207
59 184
59 420
60 494
60 13
60 39
61 437
61 10
61 71
62 335
62 126
62 239
63 319
63 300
63 133
64 255
64 177
64 201
65 29
65 133
65 282
66 133
66 434
66 52
67 474
67 72
67 341
68 406
68 493
68 69
69 429
69 361
69 341
70 494
70 187
70 345
71 89
71 211
71 230
72 38
72 205
72 20
73 408
73 284
73 151
74 44
74 352
74 72
75 392
75 284
75 487
76 136
76 352
76 477
77 56
77 352
77 164
78 241
78 492
78 330
79 375
79 178
79 372
80 458
80 221
80 327
81 364
81 386
81 418
82 97
82 486
82 182
83 211
83 51
83 433
84 137
84 427
84 239
85 34
85 402
85 230
86 0
86 462
86 273
87 276
87 195
87 476
88 414
88 483
88 309
89 204
89 421
90 265
90 262
90 325
91 276
91 86
91 460
92 349
92 444
92 69
93 22
93 246
93 18
94 139
94 481
94 16
95 111
95 377
95 370
96 24
96 124
96 384
97 151
97 352
97 132
98 14
98 57
98 134
99 123
99 418
99 409
100 329
100 422
100 216
101 411
101 150
101 484
102 191
102 474
102 360
103 472
103 97
103 392
104 288
104 324
104 60
105 24
105 263
105 493
106 259
106 328
106 10
107 257
107 105
107 230
108 230
108 94
108 291
109 7
109 187
109 298
110 490
110 223
110 12
111 476
111 55
111 230
112 174
112 411
112 39
113 293
113 464
113 276
114 319
114 201
114 268
115 77
115 174
115 430
116 282
116 122
116 257
117 144
117 440
117 53
118 425
118 454
118 368
119 359
119 216
119 76
120 121
120 29
120 393
121 312
121 20
121 123
122 479
122 465
122 143
123 127
123 190
123 113
124 271
124 211
124 67
125 297
125 165
125 146
126 112
126 20
126 310
127 259
127 464
127 54
128 173
128 205
128 365
129 182
129 473
129 203
130 414
130 118
130 334
131 388
131 57
131 95
132 44
132 291
132 327
133 132
133 259
133 436
134 226
134 65
134 117
135 270
135 170
135 220
136 362
136 234
136 232
137 148
137 136
137 168
138 243
138 336
138 146
139 275
139 436
139 341
140 421
140 174
140 375
141 54
141 288
141 158
142 176
142 283
142 117
143 307
143 151
143 114
144 16
144 90
144 419
145 152
145 458
145 459
146 116
146 193
146 101
147 94
147 134
147 387
148 459
148 244
148 87
149 96
149 36
149 25
150 218
150 488
150 6
151 360
151 448
151 363
152 38
152 31
152 263
153 29
153 235
153 219
154 70
154 210
154 458
155 412
155 249
155 136
156 234
156 344
156 22
157 400
157 78
157 192
158 22
158 395
158 166
159 432
159 376
159 382
160 341
160 222
160 191
161 192
161 115
161 52
162 303
162 384
162 263
163 253
163 386
163 286
164 313
164 199
164 404
165 2
165 405
165 113
166 441
166 391
166 313
167 75
167 219
167 132
168 123
168 249
168 342
169 489
169 153
169 244
170 193
170 80
170 433
171 59
171 98
171 138
172 153
172 60
172 443
173 104
173 152
173 100
174 373
174 24
174 249
175 296
175 67
175 447
176 29
176 4
176 179
177 219
177 308
177 196
178 294
178 50
178 304
179 37
179 209
179 54
180 105
180 17
180 163
181 68
181 227
181 136
182 350
182 14
182 9
183 462
183 465
183 462
184 23
184 281
184 23
185 47
185 148
185 48
186 125
186 165
186 325
187 224
187 279
187 391
188 349
188 322
188 50
189 28
189 159
189 45
190 257
190 342
190 61
191 342
191 412
192 434
192 411
192 44
193 143
193 121
193 148
194 300
194 218
194 183
195 369
195 297
195 391
196 307
196 179
196 139
197 319
197 293
197 62
198 390
198 297
198 389
199 303
199 191
199 450
200 39
200 250
200 372
201 447
201 414
201 485
202 342
202 53
202 9
203 65
203 338
203 485
204 329
204 277
204 293
205 242
205 398
205 415
206 487
206 103
206 210
207 116
207 232
207 194
208 181
208 77
208 86
209 319
209 363
209 85
210 433
210 403
210 458
211 65
211 331
211 418
212 398
212 50
212 97
213 111
213 381
213 464
214 147
214 374
214 431
215 74
215 49
215 10
216 379
216 315
216 277
217 227
217 128
217 301
218 356
218 149
218 28
219 119
219 427
219 413
220 183
220 228
220 390
221 458
221 27
221 259
222 374
222 158
222 207
223 358
223 90
223 192
224 230
224 247
224 266
225 182
225 182
225 361
226 370
226 472
226 113
227 338
227 151
227 205
228 240
228 331
228 202
229 29
229 434
229 316
230 39
230 215
230 316
231 275
231 418
231 381
232 317
232 294
232 220
233 487
233 17
233 118
234 368
234 61
234 24
235 440
235 434
235 239
236 123
236 135
236 216
237 447
237 196
237 369
238 204
238 304
238 456
239 171
239 101
239 323
240 206
240 54
240 465
241 269
241 71
241 318
242 228
242 355
242 280
243 271
243 43
243 153
244 428
244 412
244 220
245 62
245 344
245 146
246 48
246 43
246 39
247 353
247 392
247 450
248 113
248 447
248 103
249 412
249 333
249 465
250 422
250 276
250 233
251 476
251 246
251 47
252 417
252 347
252 260
253 43
253 438
253 90
254 163
254 281
254 272
255 99
255 458
255 452
256 326
256 126
256 259
257 206
257 163
257 418
258 184
258 498
258 235
259 449
259 376
259 465
260 190
260 178
260 323
261 326
261 225
261 333
262 161
262 153
262 55
263 196
263 83
263 318
264 451
264 151
264 294
265 52
265 302
265 120
266 78
266 185
266 55
267 158
267 442
267 330
268 175
268 235
268 71
269 109
269 355
269 191
270 225
270 167
270 112
271 65
271 450
271 385
272 394
272 191
272 97
273 461
273 237
273 355
274 167
274 116
274 129
275 270
275 158
275 120
276 344
276 368
276 499
277 115
277 49
277 98
278 383
278 297
278 32
279 146
279 66
279 172
280 304
280 413
280 247
281 0
281 293
281 356
282 59
282 105
282 319
283 130
283 281
283 419
284 61
284 4
284 47
285 489
285 341
285 402
286 497
286 348
286 333
287 5
287 18
287 320
288 93
288 214
288 466
289 25
289 427
289 306
290 143
290 219
290 16
291 380
291 34
291 98
292 354
292 13
292 131
293 176
293 375
293 390
294 73
294 346
294 162
295 162
295 145
295 110
296 82
296 31
296 288
297 262
297 418
297 244
298 314
298 305
298 93
299 129
299 461
299 53
300 289
300 307
300 392
301 435
301 257
301 256
302 144
302 353
302 297
303 372
303 12
303 353
304 395
304 181
304 301
305 424
305 352
305 267
306 235
306 388
306 83
307 36
307 116
307 50
308 135
308 377
308 360
309 280
309 147
309 287
310 52
310 116
310 281
311 478
311 196
311 196
312 313
312 89
312 497
313 67
313 190
313 478
314 282
314 421
314 112
315 98
315 120
315 192
316 239
316 147
316 111
317 497
317 299
317 104
318 113
318 325
318 341
319 175
319 158
319 117
320 34
320 82
320 122
321 473
321 247
321 384
322 219
322 112
322 433
323 106
323 368
323 367
324 95
324 239
324 51
325 307
325 476
325 262
326 215
326 121
326 158
327 275
327 86
327 215
328 376
328 304
328 128
329 272
329 122
329 476
330 246
330 254
330 95
331 203
331 382
331 237
332 170
332 497
332 212
333 470
333 408
333 23
334 253
334 19
334 385
335 208
335 187
335 449
336 280
336 154
336 164
337 227
337 59
337 121
338 222
338 188
338 302
339 405
339 443
339 432
340 216
340 110
340 491
341 349
341 371
341 417
342 410
342 410
342 96
343 286
343 331
343 324
344 443
344 40
344 232
345 355
345 53
345 95
346 64
346 209
346 126
347 236
347 455
347 95
348 353
348 105
348 153
349 452
349 22
349 364
350 182
350 180
350 155
351 164
351 440
351 260
352 163
352 454
352 81
353 116
353 322
353 329
354 472
354 69
354 494
355 288
355 72
355 239
356 121
356 491
356 493
357 208
357 384
357 92
358 370
358 389
358 92
359 223
359 228
359 12
360 188
360 17
360 87
361 71
361 394
361 200
362 197
362 419
362 489
363 151
363 415
363 471
364 468
364 223
364 421
365 484
365 315
365 285
366 246
366 389
366 125
367 62
367 208
367 479
368 455
368 134
368 345
369 74
369 347
369 321
370 4
370 211
370 24
371 497
371 213
371 360
372 421
372 276
372 27
373 121
373 224
373 371
374 93
374 493
374 356
375 120
375 252
375 467
376 247
376 188
376 291
377 407
377 296
377 154
378 60
378 352
378 52
379 5
379 402
379 263
380 14
380 359
380 233
381 344
381 74
381 197
382 78
382 126
382 160
383 112
383 211
383 136
384 222
384 61
384 204
385 148
385 153
385 460
386 158
386 482
386 195
387 427
387 344
387 96
388 485
388 277
388 228
389 130
389 304
389 279
390 197
390 152
390 78
391 437
391 449
391 218
392 413
392 475
392 338
393 483
393 224
393 305
394 68
394 352
394 277
395 457
395 452
395 327
396 434
396 390
396 441
397 495
397 249
397 436
398 105
398 441
398 431
399 382
399 451
399 108
400 248
400 315
400 49
401 349
401 349
401 30
402 475
402 491
402 453
403 359
403 68
403 370
404 310
404 12
404 71
405 90
405 437
405 494
406 322
406 264
406 130
407 3
407 425
407 444
408 72
408 325
408 206
409 194
409 181
409 374
410 460
410 301
410 468
411 433
411 168
411 271
412 377
412 419
412 252
413 83
413 397
413 459
414 386
414 72
414 16
415 0
415 299
415 341
416 456
416 282
416 423
417 254
417 51
417 268
418 444
418 383
418 21
419 236
419 55
419 189
420 248
420 361
420 137
421 344
421 130
421 351
422 275
422 288
422 242
423 26
423 461
423 17
424 469
424 36
424 379
425 401
425 401
425 115
426 158
426 406
426 320
427 138
427 130
427 182
428 49
428 334
428 66
429 113
429 323
429 158
430 315
430 445
430 25
431 466
431 158
431 168
432 134
432 420
432 364
433 103
433 297
433 105
434 203
434 203
434 64
435 423
435 9
435 316
436 25
436 411
436 12
437 2
437 131
437 243
438 76
438 151
438 347
439 285
439 180
439 228
440 490
440 58
440 286
441 127
441 88
441 37
442 248
442 456
442 243
443 138
443 464
443 119
444 350
444 86
444 168
445 69
445 253
445 126
446 57
446 169
446 280
447 293
447 343
447 153
448 148
448 70
448 111
449 131
449 371
449 435
450 83
450 437
450 182
451 67
451 31
451 265
452 238
452 28
452 367
453 256
453 452
453 152
454 499
454 98
454 490
455 338
455 82
455 259
456 325
456 330
456 491
457 349
457 83
457 270
458 30
458 82
458 58
459 54
459 480
459 59
460 60
460 356
460 244
461 117
461 63
461 457
462 86
462 185
462 287
463 441
463 362
463 354
464 281
464 16
464 184
465 143
465 221
465 485
466 221
466 402
466 403
467 200
467 146
467 82
468 239
468 403
468 241
469 287
469 0
469 289
470 49
470 455
470 330
471 20
471 317
471 44
472 435
472 218
472 332
473 181
473 468
473 369
474 35
474 18
474 341
475 178
475 84
475 272
476 116
476 67
476 78
477 248
477 111
477 87
478 433
478 340
478 140
479 41
479 499
479 166
480 357
480 321
480 143
481 333
481 483
481 360
482 333
482 122
482 195
483 100
483 200
483 488
484 320
484 81
484 192
485 85
485 354
485 316
486 250
486 220
486 238
487 488
487 339
487 396
488 57
488 94
488 425
489 356
489 112
489 12
490 307
490 78
490 371
491 238
491 353
491 228
492 436
492 365
492 401
493 192
493 274
493 32
494 4
494 408
494 43
495 498
495 190
495 154
496 413
496 402
496 446
497 216
497 186
497 54
498 185
498 76
498 442
499 19
499 41
500 3
0 0
