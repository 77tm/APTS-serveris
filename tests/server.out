5
1 6 54 63 65 79 95 102 103 113 125 128 129 136 139 152 159 162 163 169 197 207 213 220 227 230 253 296 325 333 344 375 385 413 420 421 428 434 441 455 459 492 510 520 528 532 538 539 559 563 567 576 586 621 642 643 653 656 665 669 681 701 725 730 746 760 761 785 792 795 848 853 856 874 883 886 896 919 935 950 953 973 979 992 998 